nilearn/plotting/html_surface.py

```diff
@@ -121,7 +121,8 @@ def one_mesh_info(
                           vmax=vmax, vmin=vmin)
 
 
-def _check_mesh(mesh):
+def check_mesh(mesh):
+    """Validate type and content of a mesh."""
```

Comment:
But then we need a full docstring here, right ?

Reply:
Just specify the input type in this case? Sure I can add that though I don't think that we should start adding doc strings just because we remove the underscore.

Reply:
But this is part of the public API, isn't it ?

Reply:
depends public with respect to what
it is public to the `nilearn/plotting/html_surface.py` module but not listed in the `__all__` of `nilearn/plotting/__init__.py` so not exposed to the user

Reply:
Note that what is defined in inits under `__all__` is not necessarily what is or should be exposed to the user. Take for example everything under `_utils`. We should instead look at what is defined under `doc/modules`. I think the confusion comes from the fact that the leading underscore is a naming convention and it can be done on different levels to imply that an object should be used internally. Additional confusion comes from inconsistencies in nilearn as well as cases where it might be difficult to decide clearly how to name an object based on e.g. its location. Even though we have defined some guidelines, I find that there are quite a few cases where its not so straightforward to follow them

Reply:
I agree the guidelines are neither simple nor convenient.

Well one good thing about https://github.com/nilearn/nilearn/pull/2991 is it may help us at operationalize things a bit.

The thing that irks me is that the private / public nature function in question here has not changed, so I don't see the point of now adding an extensive doc string that was not there before.

The minimal doc string I added was just to pacify flake8.

Reply:
Sorry, I checked too superficially. You can ignore my comment.